run_mac.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+THIS_DIR="$(pwd)"
+ROS_DIR="$THIS_DIR/ros"
+
+# Re-build the code
+echo "Building the code..."
+$THIS_DIR/build.sh
+
+# Launch simulator if it's not running yet
+if pgrep "mac_sys_int" > /dev/null
+then
+    echo "Great, the simulator is already running!"
+else
+    echo "Starting the simulator..."
+    cd $THIS_DIR/ros/src/styx
+    ./unity_simulator_launcher.sh &
+    cd $THIS_DIR
+fi
+
+# Launch ROS nodes
+docker run --rm=true --tty=true --interactive=true               \
+           --volume="/tmp":"/.ros"                               \
```

Comment:
I still think that somehow the --user flag should be used, otherwise files owned by root might be created. But it that's not an issue then no problem ;)